src/plugins/intel_gpu/src/plugin/ops/space_to_batch.cpp

```diff
@@ -24,30 +24,44 @@ static void CreateSpaceToBatchOp(Program& p, const std::shared_ptr<ngraph::op::v
     std::vector<cldnn::tensor> tensor_inputs;
     tensor_inputs.reserve(3);
 
+    bool param_input = false;
     for (size_t i = 1; i < 4; ++i) {
         auto inConst = std::dynamic_pointer_cast<ngraph::op::Constant>(op->get_input_node_shared_ptr(i));
-        OPENVINO_ASSERT(inConst != nullptr, "[GPU] Unsupported parameter nodes type in ", op->get_friendly_name(), " (", op->get_type_name(), ")");
-
-        std::vector<int32_t> sizes = inConst->cast_vector<int32_t>();
-        int32_t default_size = i == 1 ? 1 : 0;
-        for (size_t s = sizes.size(); s < format.dimension(); s++) {
-            sizes.push_back(default_size);
+        if (!inConst) {
+            param_input = true;
```

Comment:
nit: Could you change the variable naming to non_constant_input? I am curious why it is "param".

Reply:
done